backend/prismaCalls.ts

```diff
@@ -93,3 +93,52 @@ export async function getMetaDataByRegEx(regEx: string): Promise<prismaSchema.Pa
         return null;
     }
 }
+
+export async function getPackage(queryID: apiSchema.PackageID): Promise<apiSchema.Package | null> {
+	// Assume this type is equivalent to the Prisma package type
+	try {
+		const packageEntry = await prisma.package.findFirst({
+			where: {
+				metadata: {
+					id: queryID,
+				},
+			},
+			include: {
+				data: true,
+				metadata: true,
+			},
+		});
+		return packageEntry as unknown as apiSchema.Package;
```

Comment:
why do you have to do it like this where you use unknown?

Reply:
type error